cmd/certificates/claims.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/pkg/certificates"
+	"storj.io/storj/pkg/cfgstruct"
+)
+
+var (
+	claimsCmd = &cobra.Command{
+		Use:   "claims",
+		Short: "Print claim information",
+		RunE:  cmdClaims,
+	}
+
+	claimsCfg struct {
+		certificates.CertServerConfig
+		Raw bool `default:"false" help:"if true, the raw data structures will be printed"`
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(claimsCmd)
+
+	cfgstruct.Bind(claimsCmd.Flags(), &claimsCfg, cfgstruct.ConfDir(defaultConfDir))
+}
+
+func cmdClaims(cmd *cobra.Command, args []string) (err error) {
+	authDB, err := claimsCfg.NewAuthDB()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		err = authDB.Close()
```

Comment:
Use `err = errs.Combine(err, authDB.Close()`, otherwise it will override the previous error return values.

Reply:
:+1: